delphi-checks-testkit/src/main/java/au/com/integradev/delphi/builders/AbstractDelphiTestFile.java

```diff
@@ -55,6 +55,8 @@ protected InputFile inputFile() {
       file.deleteOnExit();
 
       try (FileWriter fileWriter = new FileWriter(file, UTF_8)) {
+        // Prepend utf-8 BOM
```

Comment:
nit: `\ufeff` is just a codepoint, which isn't inherently UTF-8

Reply:
I guess if you want to be technical, it's not even the BOM, just a zero-width non breaking space 😄

In this case, the effect of this line is to write the UTF-8 BOM to the file, so imo the comment's not wrong (though my corresponding nit is that it is wrongly capitalised...)

Reply:
Oh no! I've been out-nitted 🥴